app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Task-List</h1>
+
+<section class="summary">
+<p><%= link_to "All Tasks", tasks_path %></p>
+<p><%= link_to "Add a Task", new_task_path %></p>
+</section>
+
+<ol class="details">
+  <% @tasks.each do |task| %>
+    <li>
+      <%= link_to task.name, task_path(task) %>: 
```

Comment:
Consider adding some style to show when a task is complete (such as a strike through). You need something in the view to show when a task is complete.